src/typeahead/typeahead.ts

```diff
@@ -57,7 +57,8 @@ export class NgbTypeahead implements OnInit,
   private _popupService: PopupService<NgbTypeaheadWindow>;
   private _positioning = new Positioning();
   private _valueChanges = new Subject<string>();
-  private _windowRef: ComponentRef<NgbTypeaheadWindow>;
+
+  windowRef: ComponentRef<NgbTypeaheadWindow>;
```

Comment:
Maybe we should introduce a public method, sth like `isOpen()`, `hasPopup()` or something? Marking `WindowRef` makes me nervous...